releasenotes/notes/otel-logs-config-refactor-8230743914aa83d6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    fix(otel): Ensures the /v1/logs path is correctly added to prevent log payloads from being dropped by the Agent when using OTEL_EXPORTER_OTLP_ENDPOINT configuration. 
+    Metrics and tracing are unaffected.
```

Comment:
```suggestion
    fix(otel): Ensures the /v1/logs path is correctly added to prevent log payloads from being dropped by the Agent when using OTEL_EXPORTER_OTLP_ENDPOINT configuration. Metrics and traces are unaffected.
```